code/game/objects/items/devices/scanners.dm

```diff
@@ -94,7 +94,7 @@ REAGENT SCANNER
 	if(!(istype(user, /mob/living/carbon/human) || ticker) && ticker.mode.name != "monkey")
 		to_chat(usr, "<span class='warning'>You don't have the dexterity to do this!</span>")
 		return
-	if(user.mind && user.mind.cm_skills && user.mind.cm_skills.medical < SKILL_MEDICAL_MEDIC)
+	if(user.mind?.cm_skills && user.mind.cm_skills.medical < SKILL_MEDICAL_MEDIC && integrated == FALSE)
```

Comment:
It would be better to substitute this `if(user.mind?.cm_skills && user.mind.cm_skills.medical < SKILL_MEDICAL_MEDIC && integrated == FALSE)` for a `medical_skill_check(SKILL_MEDICAL_MEDIC)`, and making a `/obj/item/device/healthanalyzer/integrated` instead, with that check redefined.
Or a `skill_check(skill, level)` proc.